### DISCUSSION TITLE
UUVR Unity engine upgrade request!!!

### DISCUSSION THREAD

**Discussion by jihkyus** [Raicuparta/universal-unity-vr#5] [Ideas]
Most of the games currently released are released with Unity Engine and Unreal Engine.
Currently, there are two types of PC games, and many games will be released with the Unity Engine in 2024, and I think the rai pel Unity Engine is another content in the VR world.

I know that a lot of updates are being made regarding uevr.

The Unity engine is limited to 2019, and although some of the games allowed users to experience the amazing experience of entering VR, the game screen did not progress after that.

Unity engine upgrade is urgently needed.

Developer, please upgrade the Unity engine.

**Comment by Raicuparta** [comment]
UUVR being limited to Unity 5-2019 wasn't a choice, it's just still early in development. You'll have to wait for future updates, but no prediction on when it will happen (if ever)
